### PR TITLE
Unset a couple GINKGO_TEST_ARGS in kubekins-e2e

### PR DESCRIPTION
**Pull request by fejta**
Address #30749

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/30797)

### CODE REVIEW

**Comment by fejta** [comment]
@k8s-bot node e2e test this issue #IGNORE

**Comment by fejta** [comment]
@k8s-bot unit test this issue #IGNORE

**Comment by fejta** [comment]
@k8s-bot verify this issue #IGNORE

File: hack/jenkins/dockerized-e2e-runner.sh [hunk 1]
```diff
@@ -62,5 +63,4 @@ docker run --rm=true -i \
   -e "WORKSPACE=/workspace" \
   ${KUBEKINS_SERVICE_ACCOUNT_FILE:+-e "KUBEKINS_SERVICE_ACCOUNT_FILE=/service-account.json"} \
   "${docker_extra_args[@]:+${docker_extra_args[@]}}" \
-  gcr.io/google_containers/kubekins-test:go1.6.3-docker1.9.1-rev3 \
-  bash -c "bash <(curl -fsS --retry 3 --keepalive-time 2 'https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/e2e-runner.sh')"
+  gcr.io/google_containers/kubekins-e2e:v20160817
```
Comment (ixdy) [inline hack/jenkins/dockerized-e2e-runner.sh]: I assume this has already been pushed?

File: hack/jenkins/e2e-image/Dockerfile [hunk 2]
```diff
@@ -25,10 +25,10 @@ ENV E2E_UP=true \
 
 # Customize these as appropriate
 ENV E2E_PUBLISH_GREEN_VERSION=false \
-    GINKGO_PARALLEL_NODES=25 \
-    GINKGO_TEST_ARGS="--ginkgo.focus=\[Conformance\]" \
     INSTANCE_PREFIX=jenkins-e2e \
     KUBERNETES_PROVIDER=gce
+    #GINKGO_PARALLEL_NODES=25 \
```
Comment (ixdy) [inline hack/jenkins/e2e-image/Dockerfile]: maybe add a comment explaining why these are commented out / what the plan is to uncomment?

File: hack/jenkins/dockerized-e2e-runner.sh (hunk 1 above)
Comment (fejta) [inline hack/jenkins/dockerized-e2e-runner.sh]: Yes

File: hack/jenkins/e2e-image/Dockerfile (hunk 2 above)
Comment (fejta) [inline hack/jenkins/e2e-image/Dockerfile]: I'll just drop them

**Comment by ixdy** [comment]
LGTM

**Comment by ixdy** [comment]
are you planning to replace this  hack/jenkins/dockerized-e2e-runner.sh with a `curl` next?

**Comment by fejta** [comment]
Yup that's the plan! Although depend on the state of things https://github.com/kubernetes/test-infra/pull/385 might have already done this

**Comment by ixdy** [comment]
we should probably also move the dockerfile etc to test-infra?

**Pull request merged**